lifelines/utils/DataHandler.py

```diff
@@ -0,0 +1,500 @@
+import numpy as np
+
+
+class DataHandler(object):
```

Comment:
Should this live in lifelines? I wonder if this library should perform data transformations, or we let the user do any preprocessing before invoking lifelines (In my notes, I often rely on patsy/pandas/external libs).